cleanup/user-cluster.yml

```diff
@@ -92,4 +92,20 @@ rules:
   verbs:
   - list
   - get
-  - delete
\ No newline at end of file
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - list
```

Comment:
I tried this with just granting "get" on all services, and it worked the same. Could we not include "list" if we can avoid it? Does it also work the same on your end?

Reply:
yes, that true. I will open a new PR targeting to 2.9...